.github/workflows/code-nightly.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         run: |
           export LEEWAY_WORKSPACE_ROOT=$GITHUB_WORKSPACE
 
-          codeHeadCommit=$(curl -H 'Accept: application/vnd.github.VERSION.sha' https://api.github.com/repos/gitpod-io/openvscode-server/commits/gp-code/main)
+          codeHeadCommit=$(curl -H 'Accept: application/vnd.github.VERSION.sha' https://api.github.com/repos/gitpod-io/openvscode-server/commits/gp-code/release/1.84)
```

Comment:
@filiptronicek  If I remember correctly, after the stable release, our nightly should point to the main.

(Point to release branch before release is to make users with latest editor to test with pre-stable one first)

Reply:
Roger, we just forgot to update it this time in-time haha